src/store/messages/saga.ts

```diff
@@ -44,6 +50,15 @@ export function* fetch(action) {
   );
 }
 
+export function* send(action) {
+  const { channelId, message, mentionedUser } = action.payload;
+
+  yield call(sendMessagesByChannelId, channelId, message, mentionedUser);
+
+  yield call(fetch, { payload: { channelId } });
```

Comment:
i think instead of re-fetching messages we probably want to just add the message directly to the channel state. we can follow that up by fetching messages, but i think just fetching will be too slow of a user experience, when we already have the message that we are adding.

Reply:
I agree, but for our case we only have the message text, we miss  the other messages properties.

Reply:
Yes, the endpoint just return the status of request, i'll create those tasks for that.